katalog/loki-distributed/deploy.yml

```diff
@@ -477,7 +477,7 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
         - name: compactor
-          image: docker.io/grafana/loki:2.7.3
+          image: docker.io/grafana/loki:2.8.0
```

Comment:
compactor's image is the only one being bumped? the other component's image remains the same?

Reply:
I will test and check it now